Felienne/hedy
# Changes for-teachers page, give manual a dedicated page

**Description**
In this PR we clean-up the for teachers page a bit by making a dedicated page for the teacher manual. Making it more visible and giving us more freedom on the styling of the manual. For now we merely copied the content and made some minor changes to the `common mistakes` section. But, this should be a good starting point to work from and improve the manual.

**Fixes**
This PR fixes #3153.

**How to test**
Log in and make sure you are a teacher. Verify the new menu option on the dropdown menu. Also verify that the manual is content-wise identical to before, only the location and styling is changed.
